x-pack/plugins/transform/public/app/mount_management_section.ts

```diff
@@ -93,7 +94,11 @@ export async function mountManagementSection(
     contentManagement,
   };
 
-  const unmountAppCallback = renderApp(element, appDependencies, isServerless);
+  const enabledFeatures: TransformEnabledFeatures = {
+    showNodeInfo: !isServerless,
+    showFrozenDataTierChoice: !isServerless,
```

Comment:
I've followed your lead with these variable names in my [ML PR](https://github.com/elastic/kibana/pull/166651).
but I'm wondering about whether we actually need two separate variables for this.
Is tier info not the same as node info?
The flag is to control whether or not we show any reference to nodes, and tiers are collections of nodes.

Reply:
Updated here [`d5441b6` (#166622)](https://github.com/elastic/kibana/pull/166622/commits/d5441b61dcb160c1ebdc5225ff29fc0896e076ad)